source/common/upstream/subset_lb.h

```diff
@@ -73,7 +73,7 @@ class SubsetLoadBalancer : public LoadBalancer, Logger::Loggable<Logger::Id::ups
     LoadBalancerPtr lb_;
   };
 
-  // Implements HostSet::MemberUpdateCb
+  // Called by HostSet::MemberUpdateCb
```

Comment:
nit: del "Called by" to match rest of code

Reply:
That's the standard when we're implementing an interface, but is it when we're explaining the caller of a function?

Reply:
sorry nevermind!